CHANGELOG.md

```diff
@@ -52,6 +52,7 @@ For older changes see the [archived Singularity change log](https://github.com/a
 
 - Fix `GOCACHE` settings for golang build on PPA build environment.
 - Ensure `DOCKER_HOST` is honored in non-build flows.
+- Add helpful error message when remote has no library client.
```

Comment:
Let's change this a little more to distinguish it from the entry for 1.1.5.  Maybe
```
- Make the error message more helpful in another place where a remote is found to have no library client.
```